README.md

```diff
@@ -1,76 +1,231 @@
+        <li><b>Install your mods in one click</b>, just like <a href="https://github.com/Assistant/ModAssistant">Mod Assistant</a>, <a href="https://github.com/Zagrios/bs-manager">BSManager</a> will allow you to install your mods quickly and securely depending on the version you choose.</li>
+        <li><b>Customize your interface</b>, we know that some of you like to customize your interfaces to the maximum. That's why <a href="https://github.com/Zagrios/bs-manager">BSManager</a> brings you peace of mind on this point, you can change the name of the versions, their color, and you can even change the color of the logo! Of course, a dark theme has been added to protect your eyes at night.</li>
+    </ul>
+</div>
+<br>
+<div align="center">
+    <img src="resources/readme/preview.png" alt="preview">
+</div>
+<br>
+
+<!--BUILT WITH-->
+<div>
+    <h2><b>Built With</b></h2>
+    <ul>
+        <li>
+            <a href="https://electron-react-boilerplate.js.org/"><img src="https://img.shields.io/badge/-Electron%20React%20Boilerplate-black?style=for-the-badge&logo=Electron" alt="Electron React Boilerplate"></a>
+        </li>
+        <li>
+            <a href="https://github.com/SteamRE/DepotDownloader"><img src="https://img.shields.io/badge/-Depot%20Downloader-2a475e?style=for-the-badge&logo=steam" alt="Depot Downloader"></a>
+        </li>
+        <li>
+            <a href="https://rxjs.dev/"><img src="https://img.shields.io/badge/-RxJs-purple?style=for-the-badge&logo=ReactiveX" alt="RxJs"></a>
+        </li>
+        <li>
+            <a href="https://tailwindcss.com/"><img src="https://img.shields.io/badge/-Tailwind%20CSS-white?style=for-the-badge&logo=Tailwind%20CSS" alt="TailWind CSS"></a>
+        </li>
+    </ul>
+</div>
+<br>
+
+<!--CONTRIBUTING-->
+<div>
+    <h2><b>Contributing</b></h2>
+</div>
+
+<div align="center">
+    <h3><b><u>Main Developer</u></b></h3>
+    <br>
+    <a href="https://github.com/Zagrios"><img src="https://images.weserv.nl/?url=avatars.githubusercontent.com/u/40181755?v=4&h=150&w=150&fit=cover&mask=circle&maxage=7d"></a>
+    <br><br>
+    <h3><b><u>Support Developer</u></b></h3>
+    <br>
+    <a href="https://github.com/Iluhadesu"><img src="https://images.weserv.nl/?url=avatars.githubusercontent.com/u/92525749?v=4&h=150&w=150&fit=cover&mask=circle&maxage=7d"></a>
+    <a href="https://github.com/GaetanGrd"><img src="https://images.weserv.nl/?url=avatars.githubusercontent.com/u/40648115?v=4&h=150&w=150&fit=cover&mask=circle&maxage=7d"></a>
+</div>
+<div align="right">
+    [<a href="#readme-top">Return to top</a>]
+</div>
+<br>
+
+<!--USAGE-->
+<div align="center">
+    <h1> </h1>
+    <h1><b>Usage</b></h1>
+</div>
+
+<!--HOW TO INSTALL?-->
+<div>
+    <h2><b>How to install?</b></h2>
+    <ul>
+        <li>Download the latest release from <a href="https://github.com/Zagrios/bs-manager/releases">Releases</a>.</li>
```

Comment:
🟡 maybe redirect to https://github.com/Zagrios/bs-manager/releases/latest

Reply:
no tag will change

Reply:
```suggestion
        <li>Download the <a href="https://github.com/Zagrios/bs-manager/releases/latest">latest release</a> from <a href="https://github.com/Zagrios/bs-manager/releases">Releases</a>.</li>
```

that* 

my bad, I didn't notice the link

Reply:
yes